extensions/ql-vscode/src/helpers.ts

```diff
@@ -785,3 +785,24 @@ export async function* walkDirectory(
     }
   }
 }
+
+export function getFirstStoragePath() {
```

Comment:
Now that this has moved into a helpers file, do you think there's a better name for this function? Right now it doesn't really mean anything until you look at the implementation. I think JSDoc documentation would also help here.